eslint.config.js

```diff
@@ -45,6 +44,9 @@ export default [
     plugins: {
       'react-refresh': reactRefresh,
     },
+    rules: {
+      'react-refresh/only-export-components': 'warn',
```

Comment:
@williamputraintan  I found this will create a lot of warnings for share constants or functions between components or layz load,
>Fast refresh only works when a file only exports components. Move your component(s) to a separate file.

just thinking, do we need apply these changes to make 'Fast Refresh' as our common rules in developing ...

Reply:
Well, we don't need to but I saw that warning on the terminal and this helps just to identify which one causes the fast refresh to fail, happy to remove it if you found this useless.